main.cc

```diff
@@ -5,6 +5,35 @@
 
 #include "output.h"
 
+void load_test() {
```

Comment:
instead of this, do you want to tell me the format, and we can just implement the csr support for this together?
it seems like we might as well do it all at once